R/annotation-data.R

```diff
@@ -107,21 +107,23 @@ preparePromoterAnnotation <- function(txdb, species) {
   ### Retrieve promoter coordinates ###
   #####################################
```

Comment:
`promoterCoordinates` gives the coordinate of the 5' end of reduced first exons. 
`reducedExonRanges` data is integrated into the `promoterCoordinates` object by 
   - adding `intronId` data
   - storing the coordinate of the 3' end of reduced first exons (obtained with `exonEnd`)